space/space.go

```diff
@@ -87,10 +87,10 @@ type Repository interface {
 	Load(ctx context.Context, ID uuid.UUID) (*Space, error)
 	LoadMany(ctx context.Context, IDs []uuid.UUID) ([]Space, error)
 	Delete(ctx context.Context, ID uuid.UUID) error
-	LoadByOwner(ctx context.Context, userID *uuid.UUID, start *int, length *int) ([]Space, int, error)
+	LoadByOwner(ctx context.Context, userID *uuid.UUID, start *int, length *int, sort SortSpaceBy) ([]Space, int, error)
 	LoadByOwnerAndName(ctx context.Context, userID *uuid.UUID, spaceName *string) (*Space, error)
```

Comment:
Why do we only support sorting for `LoadByOwner` and not by `LoadByOwnerAndName`?

Reply:
My bad. Doesn't make any sense because LoadByOwnerAndName returns only one space ;) .

Reply:
Like we discovered in the video call, it returns only one space!